crates/typed-store/src/rocks.rs

```diff
@@ -954,6 +954,34 @@ impl<K, V> DBMap<K, V> {
                     .db_options
                     .get_ticker_count(Ticker::BloomFilterPrefixTruePositive) as i64,
             );
+        db_metrics
+            .op_metrics
+            .rocksdb_compaction_num_bytes_written
+            .with_label_values(&[&rocksdb.db_name()])
+            .set(
+                rocksdb
+                    .db_options
+                    .get_ticker_count(Ticker::CompactWriteBytes) as i64,
+            );
+        db_metrics
+            .op_metrics
+            .rocksdb_compaction_num_bytes_read
+            .with_label_values(&[&rocksdb.db_name()])
+            .set(
+                rocksdb
+                    .db_options
+                    .get_ticker_count(Ticker::CompactReadBytes) as i64,
+            );
+        db_metrics
```

Comment:
question: are these just aggregate value of the <iter|get> bytes? If so, I wonder whether we need this aggregated metrics

Reply:
It potentially is (not sure if it includes read amplification or user read bytes), however I am not sure given the documentation is not really good.